shakenfist/external_api/app.py

```diff
@@ -787,6 +793,34 @@ def error_with_cleanup(status_code, message):
 
             return db.get_instance(instance_uuid)
 
+    @jwt_required
+    def delete(self, confirm=False, namespace=None):
+        '''Delete all instances in namespace
+        '''
+        if confirm != True:
+            return error(400, "Parameter confirm is not set true")
+
+        if get_jwt_identity() == 'system':
+            if not isinstance(namespace, str):
+                return error(400, "system user must specify parameter namespace")
```

Comment:
Double quotes. Also, we are inconsistent about if error messages start with a capital or not. You don't need to fix it in this review because its pre-existing, but we should have something standard I think.

Reply:
I changed the the errors to lowercase since I was editing those lines anyway to comply with the existing standard.

(Personally I like starting them with upper case since it's easier to read amongst an error printout.)

Reply:
I have now realised that the Flask JWT package capitalises error messages. So our capitalisation is inconsistent with messages generated by that package. See [test_external_api.py#L146](https://github.com/mandoonandy/shakenfist/blob/5858fc7e1ba38623b3c2111434b76c392b755628/shakenfist/tests/test_external_api.py#L146)